src/js/burials/helpers.jsx

```diff
@@ -1,8 +1,120 @@
 import React from 'react';
+import Raven from 'raven-js';
 import moment from 'moment';
-
+import environment from '../common/helpers/environment.js';
 import { transformForSubmit } from '../common/schemaform/helpers';
 
+function checkStatus(guid) {
+  const userToken = window.sessionStorage.userToken;
+  const headers = {
+    'Content-Type': 'application/json',
+    'X-Key-Inflection': 'camel',
+  };
+
+  if (userToken) {
+    headers.Authorization = `Token token=${userToken}`;
+  }
+  return fetch(`${environment.API_URL}/v0/burial_claims/${guid}`, {
+    headers,
+    mode: 'cors'
+  })
+    .then(res => {
+      if (res.ok) {
+        return res.json();
+      }
+
+      return Promise.reject(res);
+    }).catch(res => {
+      if (res instanceof Error) {
+        Raven.captureException(res);
+        Raven.captureMessage('vets_burial_poll_client_error');
+
+        // keep polling because we know they submitted earlier
+        // and this is likely a network error
+        return Promise.resolve();
+      }
+
+      // if we get here, it's likely that we hit a server error
+      return Promise.reject(res);
+    });
+}
+
+const POLLING_INTERVAL = 1000;
+
+function pollStatus(guid, onDone, onError) {
+  setTimeout(() => {
+    checkStatus(guid)
+      .then(res => {
+        if (!res || res.data.attributes.state === 'pending') {
+          pollStatus(guid, onDone, onError);
+        } else if (res.data.attributes.state === 'success') {
+          onDone(res);
```

Comment:
This is different from VIC which queries down into the response. Both burials and pensions action want the top level of the response.

Reply:
It might be good to update the pension and burials code to be consistent with VIC. I think those changes would just be in the confirmation page?

Reply:
Probably- I'll take a look at it.

Reply:
All 3 forms fire the `schemaForm` action [`setSubmitted`](https://github.com/department-of-veterans-affairs/vets-website/blob/a879c361370236708f5123414cd0c30d70938838/src/js/common/schemaform/actions.js#L51)

This works fine for VIC which passes `res.data.attributes.response` to `setSubmitted`
<img width="360" alt="screen shot 2018-04-09 at 16 53 30" src="https://user-images.githubusercontent.com/4998130/38527373-97fea0a4-3c17-11e8-9177-bed79df70261.png">

But the pensions and burials polling endpoints don't have an `res.data.attributes.response` in dev. 
<img width="315" alt="screen shot 2018-04-09 at 16 57 17" src="https://user-images.githubusercontent.com/4998130/38527375-99ed5450-3c17-11e8-8837-9e89576451f3.png">

Theres a spot on the confirmation page for a case number so I think this is missing or we need to change the design of the confirmation page. This is the expected data based on the [ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9702#issuecomment-379082391) though.

Reply:
@lihanli I think there's a confirmation number that the backend needs to pass back, right?

Reply:
the confirmation number is passed on the form create